app/src/main/res/values/strings.xml

```diff
@@ -3,5 +3,7 @@
     <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
     <string name="fauna">Fauna</string>
-    <string name="email">Email:</string>
+    <string name="email">Email</string>
+    <string name="adgangskode">Adgangskode</string>
+    <string name="log_ind">Log ind</string>
```

Comment:
Do not use names as 'email' in your translation strings. Use a naming pattern eg:
fragment_create_account_text_input_edit_text_email_helper_text

Reply:
sure